fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/FlightElements/A32NX_Waypoint.js

```diff
@@ -141,6 +141,8 @@ class WayPoint {
             return 'CF';
         } else if (icao.charAt(0) === 'R') {
             return `${icao.substring(3, 7)}${icao.substring(9, 12).trim()}`;
+        } else if (icao.charAt(0) === 'R') {
+            return `${icao.substring(3, 7)}${icao.substring(9, 12).trim()}`;
```

Comment:
Twice the same if clause?

Reply:
Thank you! I am a derp and copied this from an old stash I had.